switchtheme.module

```diff
@@ -187,6 +201,34 @@ function switchtheme_block_view($delta = '') {
   }
 }
 
+/**
+ * returns an unordered list of active themes to switch to.
+ */
+
+function switchtheme_return_itemlist_block() {
+  $themes = list_themes();
+  $enabledThemes = []; 
+
+  $current_url = url(current_path(), array('absolute' => TRUE));
+  foreach ($themes as $key => $theme) {
+    if ($theme->status) { // if enabled
```

Comment:
See: https://api.backdropcms.org/php-standards#comments

> Comments should be on a separate line immediately before the code line or block they reference.

```suggestion
    // If enabled.
    if ($theme->status) {
```

Reply:
The code example above is to demonstrate the same comment as yours, only with proper formatting (according to our coding standards). I would suggest something this instead though:

```suggestion
    // Include only enabled themes.
    if ($theme->status) {
```